extensions/opentelemetry/opentelemetry/runtime/src/main/java/io/quarkus/opentelemetry/runtime/tracing/restclient/OpenTelemetryClientFilter.java

```diff
@@ -130,17 +130,19 @@ public void set(final ClientRequestContext carrier, final String key, final Stri
     private static class ClientSpanNameExtractor implements SpanNameExtractor<ClientRequestContext> {
         @Override
         public String extract(final ClientRequestContext request) {
-            String pathTemplate = (String) request.getProperty("UrlPathTemplate");
-            if (pathTemplate != null && pathTemplate.length() > 1) {
-                return pathTemplate;
+            StringBuilder name = new StringBuilder("HTTP");
+
+            String method = request.getMethod();
+            if (method != null && method.length() > 1) {
+                name.append(" ").append(method);
             }
 
-            String uriPath = request.getUri().getPath();
```

Comment:
I know we have the code to default to the URI there (it was kept because there was no recommendation in the first few versions, which was our implementation). I'm fine to remove it. 

In which scenarios were you getting the full URI? Maybe with the programmatic web client?